modules/tile-converter/src/deps-installer/deps-installer.ts

```diff
@@ -87,14 +87,21 @@ export class DepsInstaller {
     console.log('Installing "join-images" npm package');
     const childProcess = new ChildProcessProxy();
     const nodeDir = dirname(process.execPath);
+    let spaceEscape = '\\ ';
+    let npmBinaryName = 'npm';
+    if (process.platform === 'win32') {
+      spaceEscape = '^ ';
+      npmBinaryName = 'npm.cmd';
+    }
     await childProcess.start({
-      command: `${nodeDir}/${process.platform === 'win32' ? 'npm.cmd' : 'npm'}`,
+      command: `${nodeDir.replaceAll(' ', spaceEscape)}/${npmBinaryName}`,
```

Comment:
Why this `spaceEscape` change is related to the issue? The issue is reproduced only on Windows, but this change affects other platform as well

Reply:
According to this: https://nodejs.org/api/child_process.html#child_processspawncommand-args-options {shell: true} changes the way command is called, we now call the command in shell. And according to my tests we need to escape spaces in the path, otherwise we're getting an error. But probably we can apply {shell: true} only for windows, do you think it's a better option?

Reply:
loaders supports multiple Node versions, does this changed behavior need to be under a Node version check?